plugin/plugins/claude_companion/static/index.html

```
@@ -0,0 +1,1026 @@
+
+        <!-- 活动时间线 -->
+        <div class="card">
+            <div class="card-title"><span class="icon">📜</span> <span data-i18n="ui.timeline">活动时间线</span></div>
+            <div id="timeline-container">
+                <div class="loading"><div class="spinner"></div><div style="margin-top:10px;" data-i18n="ui.loading">加载中...</div></div>
+            </div>
+        </div>
+
+        <!-- 设置 -->
+        <div class="card">
+            <div class="card-title"><span class="icon">⚙️</span> <span data-i18n="ui.settings"></span>设置</span></div>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </span> ]

(tag-pair)